mep/people/views.py

```diff
@@ -362,7 +362,46 @@ def get_breadcrumbs(self):
             ('Home', absolutize_url('/')),
             (MembersList.page_title, MembersList().get_absolute_url()),
             (self.member.short_name, self.member.get_absolute_url()),
-            ('Membership Activities', self.get_absolute_url())
+            ('Membership', self.get_absolute_url())
+        ]
+
+
+class BorrowingActivities(LoginRequiredOr404Mixin, ListView, RdfViewMixin):
+    '''Display a list of book-related activities (borrows, purchases, gifts)
+    for an individual member.'''
+    model = Event
+    template_name = 'people/borrowing_activities.html'
+
+    def get_queryset(self):
+        # filter to requested person, then get book activities
+        return super().get_queryset() \
+                      .filter(account__persons__slug=self.kwargs['slug']) \
+                      .book_activities()
+
+    def get_context_data(self, **kwargs):
+        # should 404 if not a person or valid person but not a library member
+        # store member before calling super so available for breadcrumbs
+        self.member = get_object_or_404(Person.objects.library_members(),
```

Comment:
I wonder if this should move to the `get_queryset` method, so we bail out right away if it's not a valid slug?